src/utils/dashboardUtilFunctions.js

```diff
@@ -403,6 +403,7 @@ export function customCheckBox(data) {
       checkboxItems: transformAPIDataIntoCheckBoxData(data[mapping.api], mapping.field),
       datafield: mapping.datafield,
       show: mapping.show,
+      section: mapping.section,
```

Comment:
Can you check with Yizhen, weather this is updated in utils functions in his new upcoming PR? If not will break your changes.

Reply:
Yep. Commented on PR # 244 and requested to save these changes.